mmv1/third_party/terraform/services/iam2/resource_iam_deny_policy_test.go.erb

```diff
@@ -95,15 +95,13 @@ func TestAccIAM2DenyPolicy_iamDenyPolicyFolderParent(t *testing.T) {
 func testAccIAM2DenyPolicy_iamDenyPolicyUpdate(context map[string]interface{}) string {
```

Comment:
There are some unless version == "ga" tags at the top of this file that can be removed

Reply:
Also, you may need to change the `ProtoV5ProviderBetaFactories` used int his file to the regular GA factories